src/app/static/src/app/store/adr/actions.ts

```diff
@@ -122,20 +131,56 @@ export const actions: ActionTree<ADRState, RootState> & ADRActions = {
                         const metadata = response.data;
                         const schemas = state.schemas!;
 
-                        const uploadFiles = {
+                        const uploadFiles =  {
                             outputZip: constructUploadFile(
                                 metadata,
                                 0,
                                 schemas.outputZip,
                                 `${project.name}_naomi_outputs.zip`,
+                                `${project.name} Naomi Outputs`,
                                 "uploadFileOutputZip"),
                             outputSummary: constructUploadFile(
                                 metadata,
                                 1,
                                 schemas.outputSummary,
                                 `${project.name}_naomi_summary.html`,
+                                `${project.name} Naomi Summary`,
                                 "uploadFileOutputSummary")
-                        };
+                        } as Dict<UploadFile>;
+
+                        if (switches.adrPushInputs) {
```

Comment:
How would this feature switch be used in production ?

Reply:
When we want to switch the feature on, we just need to switch the default value from false to true. Periodically we remove old switches when features are bedded in - as here it typically just means removing a containing conditional and removing the property from the switches object